crypto/pkcs12/p12_key.c

```diff
@@ -114,10 +113,7 @@ int PKCS12_key_gen_uni(unsigned char *pass, int passlen, unsigned char *salt,
         Plen = 0;
     Ilen = Slen + Plen;
```

Comment:
Well the values normally come from trusted sources (PKCS#12 files, encrypted private keys etc) but yes it does make sense to sanity check them. Though that would apply to 1.1.0, 1.0.2 too.